assignment/assignment-tool/tool/src/java/org/sakaiproject/assignment/tool/AssignmentAction.java

```diff
@@ -17410,7 +17411,7 @@ else if(fileContentStream != null)
 							}
 						}
 
-						if(MODE_STUDENT_REVIEW_EDIT.equals(mode)) {
+						if(inPeerReviewMode) {
```

Comment:
Looking into it I think Github isn't displaying the changes correctly, it's missing the chunk expander from likes 17300 to 17375.